src/containers/Collection/Collection.tsx

```diff
@@ -172,15 +172,43 @@ export const Collection: React.SFC<CollectionProps> = ({ match, type, search, ..
   const { data: dataT } = useQuery(GET_TAGS);
   const { data } = useQuery(GET_GROUPS);
   const { data: dataUser } = useQuery(GET_USERS);
+  const { data: Collections } = useQuery(COLLECTION_QUERY, {
+    variables: {
+      filter: {},
+      opts: {
+        limit: 100,
+        offset: 0,
+        order: 'ASC',
+      },
+    },
+  });
 
   if (!data || !dataT || !dataUser) return <Loading />;
 
+  const Validate = (value: any) => {
```

Comment:
Can we change this to
```suggestion
  const validateTitle = (value: any) => {
```